src/constants/constants.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: May 05, 2018
+ * Description:
+ */
+#ifndef PROJECT_CONSTANTS_H
+#define PROJECT_CONSTANTS_H
+
+#define CLEARANCE 60 // cm
```

Comment:
what is meant by `CLEARANCE`? I would suggest renaming to be a bit more verbose, and adding a descriptive comment would also help.

Reply:
done